### PR TITLE
Stat tracker

### PR DESCRIPTION
**Pull request by sienna-kopf**
File I/O:

Sorting out the from_csv method class method, so that when it is called it creates a new instance of a stat_tracker. Also, updated the initialize method of the stat_tracker to access the elements of the hash parameter, in this case, games, teams, and game_teams. Added tests for existence of a StatTracker using the from_csv instantiation and a check for csv file attributes.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+              :teams,
+              :game_teams
+
+  def initialize(data_files)
+    @games = data_files[:games]
+    @teams = data_files[:teams]
+    @game_teams = data_files[:game_teams]
+  end
+
+  def self.from_csv(data_files)
```
Comment (dtsehl) [inline lib/stat_tracker.rb:12]: Great job on this method - the name definitely threw me for a loop the first time I saw it!

Comment (amymoller) [inline lib/stat_tracker.rb:12]: Great class / self method.  Our discussion into from_csv was extremely helpful too.

**Review by dtsehl — COMMENTED** [review]
Nice job on this! This seems like a good way to write this class and tests, and all of this passes iteration 1 too!

File: test/test_helper.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-require "simplcov"
+require "simplecov"
```
Comment (amymoller) [inline test/test_helper.rb:1]: Thank you for noticing and fixing my bad typo...

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -2,29 +2,27 @@
...
-
-
-
-
-
-
-
-
+  def setup ## instantiate using the from_csv
+    @game_path = './data/games.csv'
```
Comment (amymoller) [inline test/stat_tracker_test.rb:6]: We need to remember to go back later and take out commented out text, just when it becomes unnecessary.

**Pull request merged**